app/post.go

```diff
@@ -458,12 +458,6 @@ func (a *App) handlePostEvents(post *model.Post, user *model.User, channel *mode
 		return err
 	}
 
```

Comment:
It seems like there is some difference in the two. This query wouldn't be executed if `ThreadAutoFollow` is set to false and if it's a root post. Whereas the other one will be executed always, along with the `ThreadAutoFollow` passed to it. 

I guess the end result is the same, but I am wondering if we should add the if condition to whichever query we are keeping?